src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,15 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
```

Comment:
also think where would be the right place to have the `equals()` method and the validation.

please recollect the discussions during tutorial to help understand inheritance better.